### PR TITLE
feat: add support for draft attribute

### PR DESCRIPTION
**Pull request by sileht**
Closes: #456

### CODE REVIEW

**Comment by sileht** [comment]
arf it won't work because machine.man header doesn't receive this header.

**Comment by sileht** [comment]
?

**Comment by sileht** [comment]
@Mergifyio rebase

File: mergify_engine/actions/merge/helpers.py [hunk 1]
```diff
@@ -17,7 +17,11 @@
 
 
 def merge_report(ctxt, strict):
-    if ctxt.pull["merged"]:
+    if ctxt.pull["draft"]:
+        conclusion = None
+        title = "Draft flag need to be removed"
```
Comment (jd) [inline mergify_engine/actions/merge/helpers.py:22]:
```suggestion
        title = "Draft flag needs to be removed"
```

**Pull request merged**